### PR TITLE
[SYCLomatic] Fix migration bugs of cudaMemcpy*Array*

### PR DESCRIPTION
**Pull request by ziranzha**

### CODE REVIEW

File: clang/test/dpct/array_memory_management.cu [hunk 1]
```diff
@@ -1,326 +1,326 @@
...
+  // CHECK: dpct::dpct_memcpy(a1->to_pitched_data(), sycl::id<3>(woffset, hoffset, 0), dpct::pitched_data(data, pitch, pitch, 1), sycl::id<3>(0, 0, 0), sycl::range<3>(width, height, 1));
+  cudaMemcpy2DToArray(a1, woffset, hoffset, data, pitch, width, height, cudaMemcpyDeviceToHost);
+
+  // CHECK: dpct::async_dpct_memcpy(a1->to_pitched_data(), sycl::id<3>(woffset, hoffset, 0), dpct::pitched_data(data, pitch, pitch, 1), sycl::id<3>(0, 0, 0), sycl::range<3>(width, height, 1));
+  cudaMemcpy2DToArrayAsync(a1, woffset, hoffset, data, pitch, width, height, cudaMemcpyDeviceToHost);
+
+  // CHECK: dpct::async_dpct_memcpy(a1->to_pitched_data(), sycl::id<3>(woffset, hoffset, 0), dpct::pitched_data(data, pitch, pitch, 1), sycl::id<3>(0, 0, 0), sycl::range<3>(width, height, 1));
+  cudaMemcpy2DToArrayAsync(a1, woffset, hoffset, data, pitch, width, height, cudaMemcpyDeviceToHost, 0);
+
+  // CHECK: dpct::async_dpct_memcpy(a1->to_pitched_data(), sycl::id<3>(woffset, hoffset, 0), dpct::pitched_data(data, pitch, pitch, 1), sycl::id<3>(0, 0, 0), sycl::range<3>(width, height, 1), dpct::automatic, *s);
```
Comment (zhimingwang36) [inline clang/test/dpct/array_memory_management.cu:71]: could you double check why the whole files has been changed?  Please don't break "git blame",  keep small foot print.

Comment (ziranzha) [reply clang/test/dpct/array_memory_management.cu:71]: Change CRLF to pure ascii text. And this file is lit test, not the source code.

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Review by tomflinda — APPROVED** [review]
LGTM

**Pull request merged**